app/helpers/ux_helper.rb

```diff
@@ -0,0 +1,9 @@
+module UxHelper
```

Comment:
Can we rename this as `SettingsHelper` or `UsersHelper`, something a bit more generic? I'm saying this because `UX` also means user experience in the context of design of the user experience and it can be a bit confusing. As it's a helper that's used in what it's technically the settings maybe we can name it that.

Let me know what you think

Reply:
I think you are right, and I feel ```SettingsHelper``` is better so I renamed it already to that @rhymes